gradle.properties

```diff
@@ -8,15 +8,19 @@ eventuateMavenRepoUrl=https://dl.bintray.com/eventuateio-oss/eventuate-maven-rel
 springBootVersion=1.4.7.RELEASE
 springBootCdcVersion=2.1.1.RELEASE
 
-eventuateClientVersion=0.21.1.RELEASE
-eventuateLocalVersion=0.30.2.RELEASE
+eventuateLocalVersion=0.15.0-SNAPSHOT
```

Comment:
Does it need to use Eventuate Local? Or have things moved to Common

Reply:
I will check.